data_pipeline/surveymonkey/surveymonkey_etl.py

```diff
@@ -30,13 +30,71 @@ def get_survey_list(access_token: str) -> list:
 def get_survey_question_details(access_token: str, survey_id: str) -> list:
     headers = get_surveymonkey_api_headers(access_token)
     response = requests.get(
-        f'https://api.surveymonkey.com/v3/surveys/{survey_id}/details',
+        f"https://api.surveymonkey.com/v3/surveys/{survey_id}/details",
         headers=headers
     )
     reponse_json = response.json()
     return reponse_json
 
 
+def get_no_answer_options_for_question_json() -> dict:
+    return {
+        "id": None,
+        "text": None,
+        "type": None,
+        "weight": None
+    }
+
+
+def parse_answer_options_in_question_answer_json(
+    question_answer_response_json: dict,
+    option_key: str
+) -> dict:
+    if isinstance(question_answer_response_json[option_key], list):
```

Comment:
Maybe if the value is either a `dict` or `List[dict]` (you could declare it as the type `Union[dict, List[dict]]`),
and  the dict in both cases have the same meaning - just hat it is wrapped in list,
then you could just wrap the value in a list if it isn't, and use the same code path?

e.g.

```python
if isinstance(question_answer_response_json[option_key], list):
    answer_options = question_answer_response_json[option_key]
else:
    answer_options = [question_answer_response_json[option_key]]
return [
    {
        "id": answer_option.get("id"),
        "text": answer_option.get("text"),
        "type": answer_option.get("type"),
        "weight": answer_option.get("weight")
    }
    for answer_option in answer_options
]
```

Reply:
Thank you I will use this!